helix-term/src/commands.rs

```diff
@@ -399,7 +442,7 @@ impl<'de> Deserialize<'de> for Command {
     }
 }
 
-impl PartialEq for Command {
+impl PartialEq for MappableCommand {
     fn eq(&self, other: &Self) -> bool {
         self.name() == other.name()
```

Comment:
@sudormrfbin  this works but I'm not sure if we want to separate the Static commands from Typable commands here for the future or if we should just keep it as is for now.

Reply:
I think it's better to check for static and typable here since they are different under the hood anyways.